docs/UserGuide.md

```diff
@@ -154,6 +156,24 @@ Exits the program.
 
 Format: `exit`
 
+### Locking ClientBook : `lock`
+
+Locks ClientBook.
+
+Format: `lock [CURRENT_PASSWORD] NEW_PASSWORD`
+
+* Verifies the current password before locking ClientBook with the new password.
+* The current password field can be omitted if ClientBook is not yet locked.
+
+### Unlocking ClientBook : `unlock`
+
+Unlocks ClientBook.
+
+Format: `unlock [CURRENT_PASSWORD]`
```

Comment:
I'm a little confused about why current_password is wrapped in square brackets. Does this mean that if a password has not been set, then unlock command does not need a password?

Reply:
Yes, the original intent is for unlock command to work even without CURRENT_PASSWORD if ClientBook is not locked. However, in hindsight this is confusing for the user and I have changed CURRENT_PASSWORD to be a compulsory argument in the UG.